codecov.yml

```diff
@@ -4,4 +4,6 @@ coverage:
   - symengine/utilities/.*
   - benchmarks/.*
 
+  token: e049a7ee-c9f2-4e48-87b4-fca397853ce8
```

Comment:
Is this token supposed to be a secret?

Reply:
Good point - ideally we would instead set the env var `CODECOV_TOKEN` from a github actions secret, but github doesn't allow actions from PRs from forks to access secrets, so this wouldn't work for symengine since all PRs come from forks.
The only thing this token can be used for is uploading coverage reports, so I don't think it's a big security problem, but I agree it's not ideal to have it in plain text

Maybe another alternative to try would be just giving the two CI jobs that upload to codecov different names to see if that fixes the stale action error.